docs/guide/built-in-rules-composite.md

```diff
@@ -39,15 +39,15 @@ final class UsernameRuleSet extends Composite
 }
 ```
 
-And use it the same as one regular rule:
+And use it just like a single regular rule:
 
 ```php
 use Yiisoft\Validator\Validator;
 
 $result = (new Validator())->validate('John', new UsernameRuleSet());
 ````
 
-It can be applied for multiple attributes too:
+It can also be applied to multiple attributes:
```

Comment:
https://blog.virtualwritingtutor.com/difference-between-apply-to-vs-apply-for

> **RULE**: Use _apply_ to to say WHERE you are applying and use _apply_ for to say the PURPOSE of your application.

In this case we are indicating where it is applied, not the purpose of the application.